relays/substrate/src/main.rs

```diff
@@ -517,3 +523,67 @@ async fn estimate_message_delivery_and_dispatch_fee<Fee: Decode, C: Chain, P: En
 		Decode::decode(&mut &encoded_response.0[..]).map_err(relay_substrate_client::Error::ResponseParseFailed)?;
 	Ok(decoded_response)
 }
+
+#[cfg(test)]
+mod tests {
+	use sp_core::Pair;
+	use sp_runtime::traits::{IdentifyAccount, Verify};
+
+	macro_rules! call_for_runtime {
```

Comment:
I think we can simplify this a bit. Really, all we need is:

```rust
let call = $runtime::Call::System($runtime::SystemCall::remark(vec![]))
```

At which point we might as well just hardcode the runtime in all the call sites. I don't think a macro is worth the savings for two calls :)

Reply:
Done